docs/usage/items.md

```diff
@@ -54,18 +54,20 @@ end
-| update  | Sends update to an item                                                                            | `VirtualSwitch.update(ON)`                                   |
-| id      | Returns label or item name if no label                                                             | `logger.info(#{item.id})`                                    |
-| undef?  | Returns true if the state of the item is UNDEF                                                     | `logger.info("SwitchTest is UNDEF") if SwitchTest.undef?`    |
-| null?   | Returns true if the state of the item is NULL                                                      | `logger.info("SwitchTest is NULL") if SwitchTest.null?`      |
-| state?  | Returns true if the state is not UNDEF or NULL                                                     | `logger.info("SwitchTest has a state") if SwitchTest.state?` |
-| state   | Returns state of the item or nil if UNDEF or NULL                                                  | `logger.info("SwitchTest state #{SwitchTest.state}")`        |
-| to_s    | Returns state in string format                                                                     | `logger.info(#{item.id}: #{item})`                           |
+| Method            | Description                                                                                        | Example                                                      |
+|-------------------|----------------------------------------------------------------------------------------------------|--------------------------------------------------------------|
+| <<                | Sends command to item                                                                              | `VirtualSwitch << ON`                                        |
+| command           | alias for shovel operator (<<) - has optional arguments 'for:, on_expire:' see timed command below | `VirtualSwitch.command(ON)`                                  |
+| ensure            | Only send following command/update if the item is not already in the requested state               | `VirtualSwitch.ensure.on`                                    |
+| update            | Sends update to an item                                                                            | `VirtualSwitch.update(ON)`                                   |
+| id                | Returns label or item name if no label                                                             | `logger.info(#{item.id})`                                    |
+| linked_thing      | Returns the item's thing. If multiple, returns first. If non, returns nil.                         | `logger.info(#{item.linked_thing.uid})`                      |
+| all_linked_things | Returns an array of all things item is linked with                                                 | `logger.info(#{item.all_linked_things.map(&:uid)})`          |
```

Comment:
As these are more esoteric methods that are less often used than other methods below, should we move them to the bottom of the list? @boc-tothefuture wdyt?

Reply:
I had placed these before all the ? methods (and the related state) but I'm open to putting these wherever you folks think best.

Reply:
@ktgeek leave it as is for now unless others say otherwise. I'm merely making an uneducated suggestion. I don't know what the convention is.

Reply:
I'm fine where they are. Though I do wonder why we go through the trouble of documenting individual methods on the markdown, and _also_ commenting the source so that they render properly for yardoc (https://rdoc.info/gems/openhab-scripting). I feel like the markdown docs are fine for narrative and example, but for details of methods we should be focusing on writing good comments so they render well. I put some amount of effort into that with the types and items refactor.

Reply:
Thanks for that info. As a newbie, I didn't know where "yard docs" went and didn't know that it can be viewed so nicely on a page somewhere. Could/should the yard docs be integrated into what I consider the primary documentation which is at https://boc-tothefuture.github.io/openhab-jruby/ ? Alternatively provide a link to the rdoc.info page.